src/Progress/AirtableHandler.php

```diff
@@ -94,7 +101,17 @@ public function saveMany(array $progresses): void
         foreach ($progresses as $progress) {
             $data[] = $this->parseProgress($progress);
         }
-        $this->createRecords($data);
+        $this->updateRecords($data);
```

Comment:
See the bit on the ticket about sorting this. You want to update rows that have a row ID given by parse progress (e.g. `$progressRow['id'] !== null` to be updated and rows without an ID given by parseProgress to be created.

This needs to be done within the saveMany function - the sdk defines the interface to use so if we add an updateMany function it just won't be called